mos-platform/geos-cbm/geos_constants.h

```diff
@@ -0,0 +1,752 @@
+/***********************************************************************
+ * geos_constants.h
+ *
+ * This file contains #defines for use in GEOS applications
+ * with llvm-mos C compiler.
+ *
+ * Transcribed from geoProgrammer manual by DiSc.
+ *
+ ***********************************************************************/
+
+#ifndef GEOS_CONSTANTS_H
+#define GEOS_CONSTANTS_H
+
+/***********************************************************************
+ * Miscellaneous defines
+ ***********************************************************************/
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define LOW_BYTE(x) (*((unsigned char *)&(##x) + 0))
+#define HIGH_BYTE(x) (*((unsigned char *)&(##x) + 1))
+
+#define WORD(w) (w & 0xff), (w >> 8)
+
+#define AUX(x) #x
```

Comment:
Internal defines like this should be `#undef`ed after they are used.

Reply:
Hmm, maybe i do not get you. But if I #undef AUX the Macro using it will fail at the place where it is used.

Reply:
Nope. you're right. Sorry.